source/routing/raptor_api.cpp

```diff
@@ -1452,6 +1449,44 @@ void make_isochrone(navitia::PbCreator& pb_creator,
     }
 }
 
+void make_isochrone_distributed(navitia::PbCreator& pb_creator,
+                                RAPTOR& raptor,
+                                const type::EntryPoint& isochrone_center,
+                                const std::vector<type::EntryPoint>& stop_points,
+                                const uint64_t datetime_timestamp,
+                                bool clockwise,
+                                const type::AccessibiliteParams& accessibilite_params,
+                                std::vector<std::string> forbidden,
+                                std::vector<std::string> allowed,
+                                const type::RTLevel rt_level,
+                                int max_duration,
+                                uint32_t max_transfers) {
+    // Create datetime
+    auto const datetimes = parse_datetimes(raptor, {datetime_timestamp}, pb_creator, clockwise);
+    if (pb_creator.has_error() || pb_creator.has_response_type(pbnavitia::DATE_OUT_OF_BOUNDS)) {
+        return;
+    }
+
+    // Get stop points for departure and destination
+    auto const departures = make_map_stop_point_duration(stop_points, raptor.data.pt_data->stop_points_map);
+    auto const datetime = datetimes.front();
```

Comment:
if `datetimes` is empty -> boom!

Reply:
It is sometimes test but I forgot it here :D

Reply:
`datimes` is `request.datetimes(0)` so it would crash earlier anyway. It is the same for almost every API in kraken